lib/ansiblelint/rules/MetaMainHasInfoRule.py

```diff
@@ -4,16 +4,49 @@
 from ansiblelint.rules import AnsibleLintRule
 
 
+META_STR_INFO = [
+    'author',
+    'description',
+]
+META_INFO = META_STR_INFO + [
+    'license',
+    'min_ansible_version',
+    'platforms',
+]
+
+
+def _platform_info_errors_itr(platforms):
+    if not isinstance(platforms, list):
+        yield 'Platforms should be a list of dictionaries'
+        return
+
+    for platform in platforms:
+        if not isinstance(platform, dict):
+            yield 'Platforms should be a list of dictionaries'
+        elif 'name' not in platform:
+            yield 'Platform should contain name'
+
+
+def _galaxy_info_errors_itr(galaxy_info,
+                            info_list=META_INFO,
+                            str_info_list=META_STR_INFO):
```

Comment:
Having mutable defaults is a dangerous thing. You should either use a None-idiom or turn them into tuples.

Reply:
Absolutely right. Thanks for your advice!

I usually use a None-idiom you indicated but I feel it's a bit too much so I committed and pushed to make them into tuples.